src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -54,6 +57,137 @@ std::string_view Model::implementation::LoadStringResource(int resourceID)
     return { reinterpret_cast<const char*>(ptr), sz };
 }
 
+winrt::hstring ActionArgFactory::GetNameForAction(Model::ShortcutAction action)
+{
+    // Use a magic static to initialize this map, because we won't be able
+    // to load the resources at _init_, only at runtime.
+    static auto actionNames = []() {
+        return std::unordered_map<ShortcutAction, winrt::hstring>{

```

Comment:
Looked through all the `MustGenerate` ones that were in ActionAndArgs.cpp. `SaveSnippet` is missing from the list. Guessing it should be in there?

Reply:
No idea how that happened, fixed! Thanks for the catch